### DISCUSSION TITLE
Add dependency for `CLLocationManager`

### DISCUSSION THREAD

**Discussion by RemiBardon** [tgrapperon/swift-dependencies-additions#72] [Ideas]
There is no dependency for `CLLocationManager` yet, I am working on adding support for it (#70).

I'm also writing a more ergonomic wrapper, as presented quickly in #71.

**Comment by tgrapperon** [comment]
Hey @RemiBardon! Thanks for opening the discussion. I refrained to propose such a dependency  because I think that Brandon and Stephen have planned to rework [`composable-core-location`](https://github.com/pointfreeco/composable-core-location), and I feel that this will uncouple it from TCA and make it a standalone dependency. That's why I'm not leaning toward merging the PR right now, but I'll give it a proper look very soon anyway!
I'm also converting this issue as a discussion, as this is not a direct problem with what the library exposes right now.

**Reply by RemiBardon** [reply]
Oh that's great if they update [`composable-core-location`](https://github.com/pointfreeco/composable-core-location)!

I will use my own branch for the time being then 😉

And thanks for converting this to a discussion, I created it because your commits have `#<number>` and I overlooked that they didn't refer to issues.

**Reply by RemiBardon** [reply]
@stephencelis @mbrandonw Feel free to get inspired from what I did in #70! There may be a few backwards compatibility issues, but I'm fixing them as I find them.
